doc/cephadm/monitoring.rst

```diff
@@ -93,6 +93,33 @@ completed, you should see something like this from ``ceph orch ls``
   node-exporter      2/2  6s ago     docker.io/prom/node-exporter:latest             e5a616e4b9cf  present
   prometheus         1/1  6s ago     docker.io/prom/prometheus:latest                e935122ab143  present
 
+Configuring TLS for Grafana
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+  ceph config-key set mgr/cephadm/grafana_key -i $PWD/key.pem
+  ceph config-key set mgr/cephadm/grafana_crt -i $PWD/certificate.pem
+
+The ``cephadm`` manager module needs to be restarted to be able to read updates
+to these keys.
+
+.. code-block:: bash
+
+  ceph mgr module disable cephadm && \
+    sleep 2 && \
+    ceph mgr module enable cephadm
```

Comment:
```suggestion
  ceph orch restart mgr
```

Does this need to be reloaded on all the mgr instances? Maybe a restart would be simpler?